src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,22 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by '}
+        <UserInfo user={post.user} />
```

Comment:
Make this component with conditional rendering only if user exist

Reply:
But when I made changes, I got en error that not all post are displayed